src/components/shared/transactionsTable/index.js

```diff
@@ -10,31 +10,54 @@ import Icon from '../../toolbox/icon';
 import AccountVisual from '../../toolbox/accountVisual';
 import styles from './transactionsTable.css';
 import IconlessTooltip from '../iconlessTooltip';
+import regex from '../../../utils/regex';
+
+const windowSizeBreakpoint = 1024;
```

Comment:
We have a const file for this: [src/constants/breakpoints.js](https://github.com/LiskHQ/lisk-hub/blob/92fb9ece8bb191e93758a71c1af42b7674b3816f/src/constants/breakpoints.js)

Reply:
I replaced it with the one from the breakpoint constant file.